### PR TITLE
Example of ~/.lein/profiles.clj configuration

### PR DESCRIPTION
**Pull request by mikebohdan**
A simple example of the configuration to make usage of this library even easier.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -137,6 +137,13 @@ the working directory with the contents:
 {:drawbridge {:http-headers {:Authorization "Bearer <JWT token>"}}}
 ----
 
+Also, for using connecting repl via endpoint make sure that you added
+following line to the `~/.lein/profiles.clj`
+[source,clojure]
+---
+{:user {:plugins [[nrepl/drawbridge "0.2.1"]]}}
```
Comment (bbatsov) [inline README.adoc:144]: Note that you can also add this to your project. It doesn't necessarily have to be in the profiles.

File: README.adoc [hunk 2]
```diff
@@ -137,6 +137,13 @@ the working directory with the contents:
 {:drawbridge {:http-headers {:Authorization "Bearer <JWT token>"}}}
 ----
 
+Also, for using connecting repl via endpoint make sure that you added
```
Comment (bbatsov) [inline README.adoc:140]: It seems to me that this should be in "Usage", maybe under a "Setup" sub-section.

File: README.adoc (hunk 1 above)
Comment (mikebohdan) [reply README.adoc:144]: Thanks! I'll add it too.

**Comment by bbatsov** [comment]
Hmm, after merging this I remembered that drawbridge doesn't actually bundle a Lein plugin, so these instructions are misleading. It's just an ordinary library.

**Pull request merged**